scripts/csv2json.py

```diff
@@ -150,5 +152,20 @@ def recursive_mean(data_dict):
 # For each dict leaf, convert the list of values to its mean.
 recursive_mean(data_dict)
 
-with open(args.output_file, "w") as jsonfile:
-    json.dump(dict(data_dict), jsonfile, indent=4)
+if not os.path.exists(args.output_dir):
+    os.makedirs(args.output_dir)
+
+characters_to_replace = " :()"
+for key in data_dict.keys():
+    filename = key
+
+    # Remove repeated spaces.
+    filename = re.sub(" +", " ", filename)
+
+    for character in characters_to_replace:
+        filename = filename.replace(character, "_")
```

Comment:
The HUC files end up with a trailing `_` -- not sure if that's coming in here, or if it's something we care much about or not.

Reply:
IMO it's not worth worrying about the trailing `_` in JSON filenames since users will never see the filenames unless they dig into their browser's network developer tools, and by then the filenames also have a hash added to them, so even goofier looking!